scriptworker/constants.py

```diff
@@ -44,6 +54,16 @@
     "poll_interval": 10,
     "sign_key_timeout": 60 * 2,
 
+    "reversed_statuses": frozendict({
+        -11: STATUSES['intermittent-task'],
+        -15: STATUSES['intermittent-task'],
```

Comment:
I'm confused about these values. How different are -11 and -15?

Reply:
We were getting -11 for segfaults, and we get -15 when we kill the task.